### ISSUE TITLE
zsteg executable won't run with 'require' error

### ISSUE THREAD

**Issue by yutanagano** [zed-0xff/zsteg#22]
Hello,

I am not familiar with Ruby, so I may be missing something very obvious, but I think I have an issue, which is specific to zsteg.

After installing zsteg to my machine with `gem install zsteg`, the zsteg executable will not run. I get the following error:

```
<internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- prime (LoadError)
	from <internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/yutanagano/.local/share/gem/ruby/gems/zsteg-0.2.9/lib/zsteg/extractor.rb:1:in `<top (required)>'
	from <internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/yutanagano/.local/share/gem/ruby/gems/zsteg-0.2.9/lib/zsteg.rb:6:in `<top (required)>'
	from <internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/yutanagano/.local/share/gem/ruby/gems/zsteg-0.2.9/bin/zsteg:6:in `<top (required)>'
	from ./zsteg:25:in `load'
	from ./zsteg:25:in `<main>'
```

I thought at first that maybe something is wrong with the Ruby install on my machine and tried installing other gems, and they worked fine. That's why I'm posting my issue here.

I am on a Fedora Linux (v36) machine.
Ruby version: `ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux]`
Gem version: `3.3.7`

Thank you!

**Comment by lSANCHOl** [comment]
I can confirm I am getting the same error message on:
Fedora (v36)
Ruby version: `ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux]`
Gem version: `3.3.7`

**Comment by zed-0xff** [comment]
fixed in 0.2.11
